database/src/test/kotlin/no/nav/su/se/bakover/database/dokument/DokumentPostgresRepoTest.kt

```diff
@@ -40,6 +40,7 @@ internal class DokumentPostgresRepoTest {
                     vedtakId = etVedtak.id,
                     revurderingId = enRevurdering.id,
                     bestillBrev = false,
+
```

Comment:
Savner en med og uten status test :)

Reply:
Egentlig er de feltene kun populeres ved henting av data. Hvis man legger til en verdi ved `lagre` så vil faktisk ingenting skje. Sånn sett kan det argumenteres for at jeg kunne lagd en ny datatype som kun brukes ved uttrekk, men ser ut som at bestillBrev er gjort på samme måte bortsett fra at det kun er et felt som brukes ved lagring (og ikke hent)?

Reply:
Test: 386db7b16e1484c685cbfc48e1cd7125ce08c511